src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import './DropdownMenu.scss';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson: Person | null;
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownMenu: FC<Props> = ({
+  people,
+  selectedPerson,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => {
+          const { name, sex } = person;
+
+          return (
+            <div
+              key={person.slug}
+              className={cn('dropdown-item', {
+                'has-background-grey-lighter': selectedPerson === person,
+              })}
+              data-cy="suggestion-item"
+              onClick={() => onSelect(person)}
```

Comment:
This is a performance issue. You are creating a new function every time the component renders. This is not recommended. Instead, create a function inside the component that accepts a person as an argument and call this function in your onClick handler. This way, a new function is not created on every render.